packages/front-end/components/Tooltip.tsx

```diff
@@ -4,18 +4,24 @@ import { MdInfoOutline } from "react-icons/md";
 // todo: add support for variable positioning instead of just bottom.
 
 interface Props extends HTMLAttributes<HTMLDivElement> {
-  text: string;
+  text?: string;
+  ToolTipComp?: JSX.Element;
```

Comment:
What I've done elsewhere is just have a single prop and make the type `string | ReactElement`